app/renderer/components/navbar.js

```diff
@@ -97,6 +101,24 @@ module.exports = {
                 default:
                     this.AppStatus.events.emit(selection);
             }
+        },
+        addShortcuts() {
```

Comment:
2 improvements here:
1. Could you please wrap all Mousetrap usage in a separate file and use that interface here instead please?
2. `$('#navBarMenu').click()` Is a jquery event, for now I'm wrapping all these events in the file `materialize.js` to avoid high coupling and make future refactorings easier. Could you please wrap this event as well?